build.sh

```diff
@@ -0,0 +1,9 @@
+function error {
+    echo "Error: $1"
+    exit 1
+}
+
+julia src/generate.jl spmv rmat || error
```

Comment:
Which julia is this? Lets add some of the boilerplate from https://github.com/willow-ahrens/Finch.jl/blob/main/.github/workflows/CI.yml to setup julia with a known version